src/components/application_manager/test/commands/mobile/scrollable_message_test.cc

```diff
@@ -180,15 +180,16 @@ TEST_F(ScrollableMessageRequestTest, Init_CorrectTimeout_SUCCESS) {
       mobile_apis::InteractionMode::MANUAL_ONLY;
   EXPECT_EQ(kDefaultTimeout_, command_->default_timeout());
   command_->Init();
-  EXPECT_EQ(kTimeOut, command_->default_timeout());
+  const uint32_t result_timeout = kTimeOut + kDefaultTimeout_;
+  EXPECT_EQ(result_timeout, command_->default_timeout());
 }
 
 TEST_F(ScrollableMessageRequestTest, Init_CorrectTimeout_UNSUCCESS) {
   (*msg_)[msg_params][interaction_mode] =
       mobile_apis::InteractionMode::MANUAL_ONLY;
   EXPECT_EQ(kDefaultTimeout_, command_->default_timeout());
   command_->Init();
-  EXPECT_EQ(kTimeOut, command_->default_timeout());
+  EXPECT_NE(kTimeOut, command_->default_timeout());
```

Comment:
@AByzhynar propose to remove this test case as redundant

Reply:
@AKalinich-Luxoft Removed in dd9e6b45cd08a6c0308652adef406466a796a09c